parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -353,6 +362,12 @@ try
         New-CsiProxyService -CsiProxyPackageUrl $global:CsiProxyUrl -KubeDir $global:KubeDir
     }
 
+    if ($global:EnableSecureTLSBootstrapping) {
+        $clientPath = [Io.path]::Combine("$global:KubeDir", "tls-bootstrap-client.exe")
+        Write-Log "Secure TLS Bootstrapping is enabled, downloading bootstrap client to $clientPath if needed..."
+        Get-SecureTLSBootstrapClient -BootstrapClientPath $clientPath -BootstrapClientDownloadUrl $global:SecureTLSBootstrapClientDownloadURL
+    }
+
     if ($global:TLSBootstrapToken) {

```

Comment:
Could both EnableSecureTLSBootstrapping and TLSBootstrapToken be true?

Reply:
How do we know which actually works If both of them exist? Could we set $global:TLSBootstrapToken to false automatically when $global:EnableSecureTLSBootstrapping is true? I suppose that $global:EnableSecureTLSBootstrapping will be behind of a toggle so we can rollout it per sub or by region and easily rollout it back.

Reply:
it's mentioned in the design doc, but we primarily plan on determining success rates based on the guest-agent telemetry events generated by securetlsbootstrap.ps1 (e.g. Logs-To-Events/As-Event)